.circleci/config.yml

```diff
@@ -141,6 +136,43 @@ jobs:
             git push --force origin $BRANCH
             hub pull-request -m "[CI] Update locales" || true
 
+  update-dependencies:
+    <<: *defaults
+    steps:
+      - checkout
+      - *restore-env-cache
+      - *setup-env
+      - *add-token-to-remote
+      - run:
+          name: update dependencies and make pull request
+          environment:
+            BRANCH: circleci-update-dependencies
+          command: |
+            set -xe
+            git config credential.helper 'cache --timeout=120'
+            git config user.email "ci@foodsaving.world"
+            git config user.name "CircleCI"
+            git branch --delete --force $BRANCH || true
+            git checkout -b $BRANCH
+            pip-compile --upgrade
+            git add requirements.txt
+            pushd mjml
+            yarn upgrade --latest
+            git add package.json yarn.lock
+            popd
+            git commit -m "[CI] update dependencies" || exit 0
+            git push --force origin $BRANCH
+            hub pull-request -m "[CI] Update dependencies" -F - <<EOL || true
```

Comment:
This seems confusing, either use `-m` or `-F`: https://hub.github.com/hub-pull-request.1.html

Reply:
Yup, good catch! Will update.